### DISCUSSION TITLE
Enable BitLocker on Windows 10 Home

### DISCUSSION THREAD

**Discussion by Mariusica** [Fleex255/PolicyPlus#81] [Q&A]
Hello everyone. I'v try'd to activate policy editor on windows 10 home edition, with the release of Fleex255 and appears the platform. I'm trying to enable the BitLocker service, figures like enabled but don't appear anyware on the control panel or else. Can anyone help me?
Thank's

**Comment by Fleex255** [comment]
While many policy settings work on Home editions if applied to the Registry (which is how Policy Plus works internally), some Windows components are simply not present on Home editions. In those cases, editing policy/Registry settings has no effect. My impression is that the BitLocker GUI is not present on Home editions.

You can try to enable BitLocker via other means. This is independent of policy settings. **Caution! I have not tested this. Make sure you have backups to restore from.** [This Super User question](https://superuser.com/q/1590289/380318) has some possible answers.

**Comment by Mariusica** [comment]
Thank you very much Ben.
I''ve try'd it but doesn't work on my version of windows, it comes back
with negative value. At least now i know that doesn't work for me.

Best regards

Adi

Il giorno sab 21 ott 2023 alle ore 12:21 Ben Nordick <
***@***.***> ha scritto:
